robosuite/controllers/README.md

```diff
@@ -0,0 +1,43 @@
+# Controllers  
+  Controllers are used to determine the type of high-level control over a given robot arm. While all arms are directly controlled via their joint torques, the inputted action space for a given environment can vary depending on the type of desired control. Below, a list of supported controllers and their respective action dimensions are listed.
+  |   Controller   					| 			Action Space (not including gripper)    |  
+| :-----------------------------------------------: | :--------------------------------:|  
+|    End Effector Position Orientation (EE_POS_ORI) |    6 (x, y, z, r, p, y)          	|  
+|    End Effector Position (EE_POS) 				|    3 (x, y, z)      				|  
+|    End Effector Inverse Kinematics (EE_IK)  		|    7 (x, y, z, i, j, k, w) 		|  
+|    Joint Position (JOINT_IMP)    					|    n (robot joint dimension)      |  
+|    Joint Velocity (JOINT_VEL)    					|    n (robot joint dimension)      |  
+|    Joint Torque (JOINT_TORQUE)   					|    n (robot joint dimension)      |  
+
+## Configurations
+The [config directory](robosuite/controllers/config) provides a set of default configuration files that hold default examples of parameters relevant to individual controllers. Note that when creating your controller config templates of a certain type of controller, the listed parameters in the default example should be specified accordingly.
+
+Below, a brief overview and description of each subset of controller parameters are shown:
+
+#### General Controller Settings  
+* `'type'`: Type of controller to control. Can be `'EE_POS_ORI'`, `'EE_POS'`, `'EE_IK'`, `'JOINT_IMP'`, `'JOINT_VEL'`, or `'JOINT_TORQUE'`
```

Comment:
What controller settings are mandatory and what are optional? And what are their default values?

Reply:
As mentioned in the Configuration section on line 13, the default templates provided in the `config` folder describe both the default values and the required parameters that should be specified.

There are additional default config files for each robot arm -- added a blurb that mentions that these specific default jsons are used by default for each robot, respectively